RELEASE_NOTES.md

```diff
@@ -1,5 +1,124 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes/> for the official Chef release notes.
 
+# What's New in 16.6
+
+## pwsh Support
+
+We've updated multiple parts of the Chef Infra Client to fully support Microsoft's `pwsh` in addition to our previous support for `PowerShell`.
```

Comment:
Probably good to add this clarification:

Microsoft's `pwsh` -> Microsoft's `pwsh` (commonly known as PowerShell Core)

Reply:
yep let me add that